test/e2e/secret_test.go

```diff
@@ -336,6 +336,16 @@ var _ = Describe("Podman secret", func() {
 		Expect(session.OutputToStringArray()).To(HaveLen(1))
 	})
 
+	It("podman secret rm --ignore", func() {
+		remove := podmanTest.Podman([]string{"secret", "rm", "non-existent-secret"})
+		remove.WaitWithDefaultTimeout()
+		Expect(remove).Should(Not(Exit(0)))
+
+		ignoreRm := podmanTest.Podman([]string{"secret", "rm", "--ignore", "non-existent-secret"})
+		ignoreRm.WaitWithDefaultTimeout()
+		Expect(ignoreRm).Should(Exit(0))
```

Comment:
Similarly, this should check that `ErrorToString()` is empty. This is lower priority than the positive error check above.